docs/src/windows.md

```diff
@@ -44,26 +24,45 @@ curl https://pyenv.run | bash
 ```
 
 - Edit ~/.bashrc (ex: ```nano ~/.bashrc```) and add this code at the end (replace {USERNAME} with your Windows session name):
+
 ```
 export PATH="/home/{USERNAME}/.pyenv/bin:$PATH"
 eval "$(pyenv init -)"
 eval "$(pyenv virtualenv-init -)"
 ```
 
-- Install Python and its libraries:
+- Install Python 3 and its libraries:
+
 ```
-pyenv install 3.6.8
 pyenv install 3.7.2
-pyenv global 3.6.8
-pip install pip --upgrade
-pip install pipenv
 pyenv global 3.7.2
 pip install pip --upgrade
 pip install pipenv
 ```
 
 - Then enter the [commands](#commands) below to build and run Erdapfel
 
+<!--
```

Comment:
Instead of commenting it, which adds useless lines of doc to maintain, I would just explain briefly that Windows 7/8 isn't "officially" supported and not recommended, and if you *really* want to run Erdapfel on it you'll need a Linux VM. No need to dive into VM config details.

It could be nice hovewer to explain, for example at the beginning, why Erdapfel doesn't work natively on Windows and always requires Linux virtualization…

Reply:
sure